src/Envoi.php

```diff
@@ -167,7 +167,7 @@ public static function markdown($envMetaPath = self::DEFAULT_META_FILE_NAME, $ma
         $meta = self::metaFromYamlFile($envMetaPath);
 
         if (count($meta) === 0) {
-            return false;
+            throw new \UnexpectedValueException("Expected there to be metadata in the meta file \"{$envMetaPath}\", but found none.");
```

Comment:
Throw an exception so that there is only one meaning of a `false` return value (markdown was not written to the file).